JS/Lesson35/onur/my-app/src/Modules/user/UserProvider.tsx

```diff
@@ -0,0 +1,65 @@
+import { useReducer, createContext, useContext } from 'react';
+import type { ReactNode, Dispatch } from 'react';
+
+interface User {
+    loggedIn: boolean;
+}
+
+export const initialUserState: User = { loggedIn: false };
+
+export enum UserActionType {
+    LOG_IN = 'logIn',
+    LOG_OUT = 'logOut',
+    SIGN_UP = 'SignUp'
```

Comment:
Consistency is important :) 
```suggestion
    SIGN_UP = 'signUp'
```